src/controls/SelectTeamChannelPicker/ISelectTeamChannelPickerProps.ts

```diff
@@ -0,0 +1,13 @@
+import { WebPartContext } from "@microsoft/sp-webpart-base";
+import { ExtensionContext } from "@microsoft/sp-extension-base";
+import { IBasePickerStyles, IPickerItemProps, ISuggestionItemProps, ITag } from "office-ui-fabric-react";
```

Comment:
Please, use specific imports, otherwise you're referencing the whole OUIFR and increase the size of our lib.